### PR TITLE
Feature/23/delete posts

### PR DESCRIPTION
**Pull request by neobay991**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -7,19 +7,46 @@ def create
...
     @user = User.find(session[:current_user_id])
     @posts = Post.where(user_id: @user.id)
   end
 
+  def destroy
+    @user = User.find(session[:current_user_id])
+    @post = Post.find(params[:id])
+    @post.destroy
+    post_delete_notice
+    redirect_back fallback_location: request.referrer
```
Comment (jeff1108) [inline app/controllers/posts_controller.rb:24]: What does it do?????????

File: spec/features/user_can_edit_posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    click_button "Update"
+    expect(page).to have_text("My changed post")
+  end
+
+
+  scenario "A user can update post from the user profile" do
+    click_link("View your profile")
+    click_link("Update")
+    fill_in :message, with: "My changed post"
+    click_button "Update"
```
Comment (jeff1108) [inline spec/features/user_can_edit_posts_spec.rb]: expect page to have content "Your post has been updated?"

**Review by jeff1108 — CHANGES_REQUESTED** [review]

File: spec/features/user_can_edit_posts_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  scenario "A user can delete post from the user homepage" do
+    click_link("Update")
+    fill_in :message, with: "My changed post"
+    click_button "Update"
+    expect(page).to have_text("My changed post")
+  end
+
+
+  scenario "A user can update post from the user profile" do
+    click_link("View your profile")
```
Comment (jeff1108) [inline spec/features/user_can_edit_posts_spec.rb]: Add web_helper.rb??

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (elishkaflint) [reply app/controllers/posts_controller.rb:24]: redirects back to the page where the request was made

File: spec/features/user_can_edit_posts_spec.rb (hunk 3 above)
Comment (elishkaflint) [reply spec/features/user_can_edit_posts_spec.rb]: done

**Review by jeff1108 — APPROVED** [review]

File: spec/features/user_can_edit_posts_spec.rb (hunk 2 above)
Comment (elishkaflint) [reply spec/features/user_can_edit_posts_spec.rb]: done

**Comment by jeff1108** [comment]
:shipit:

**Pull request merged**